.travis.yml

```diff
@@ -13,12 +13,17 @@ env:
     - STACK_DATA_SLOW_TESTS=1
     - COVERALLS_PARALLEL=true
 
+before_install:
+  - pip install --upgrade coveralls setuptools>=44 setuptools_scm>=3.4.3 pep517
+
 install:
-  - pip install coveralls
-  - pip install .[tests]
+  - python3 -m pep517.build .
+  - ls -l
+  - mv ./dist/*.whl ./dist/stack_data-0.CI-py3-none-any.whl
+  - pip install "./dist/stack_data-0.CI-py3-none-any.whl[tests]"
 
 script:
-  - coverage run --include='stack_data/*' -m pytest
+  - coverage run --branch --include='stack_data/*' -m pytest --junitxml=./rspec.xml
```

Comment:
Why these changes?

Reply:
Just improvements.
1. branch coverage is more sensible metric than line coverage, so why not to compute it too?
2. `--junitxml=./rspec.xml` is just a reminder that it is possible to generate a report. Appveyor and GitLab CI can use them to draw a list of passed and failed tests. IDK about GH Actions. So when you migrate to something supporting reading the list, it makes sense to register this artifact to be used.